pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -711,6 +751,10 @@ func (c InstallerController) sync() error {
 		return nil
 	}
 
+	if err := c.ensureCerts(); err != nil {
+		return err
```

Comment:
can we log the error here? (V(4)) ?

Reply:
iow. "why my installer pods are not starting"

Reply:
> iow. "why my installer pods are not starting"

level above logs